pkg/codec/vaapi/vp8.go

```diff
@@ -100,6 +101,8 @@ type encoderVP8 struct {
 
 	rate *framerateDetector
 
+	forceKeyFrame atomic.Bool
```

Comment:
Need to bump minimal supported go version to 1.19
https://github.com/pion/mediadevices/blob/b4770e5fbf35214a92cb0a5fc335283893bd9fa8/go.mod#L3